biggus/__init__.py

```diff
@@ -2560,6 +2726,30 @@ def sub(a, b):
     return _Elementwise(a, b, np.subtract, np.ma.subtract)
 
 
+def multiply(a, b):
+    """
+    Return the elementwise evaluation of `a * b` as another Array.
+
+    """
+    return _Elementwise(a, b, np.multiply, np.ma.multiply)
+
+
+def divide(a, b):
+    """
+    Return the elementwise evaluation of `a / b` as another Array.
+
+    """
+    return _Elementwise(a, b, np.divide, np.ma.divide)
```

Comment:
I'm just going to replace ``np.divide`` with ``np.true_divide`` and completely ignore the prospect of floor division. Is that OK?